mail/send.go

```diff
@@ -271,3 +271,200 @@ func SendMail(obj MailRequest) {
 
 	log.Println("[SendMail] end ----------------------------------------------")
 }
+
+func SendMailV2(obj MailRequest) error {
+	log.Println("[SendMail] start --------------------------------------------")
+	var (
+		username = os.Getenv("EMAIL_USERNAME")
+		password = os.Getenv("EMAIL_PASSWORD")
+		from     = AddressAnna
+		file     []byte
+		tpl      bytes.Buffer
+		err      error
+	)
+
+	switch os.Getenv("env") {
+	case "local":
+		file, err = os.ReadFile("../function-data/dev/mail/mail_template.html")
+	case "dev":
+		file, err = lib.GetFromStorageV2("function-data", "mail/mail_template.html", "")
+	case "prod":
+		file, err = lib.GetFromStorageV2("core-350507-function-data", "mail/mail_template.html", "")
+	}
```

Comment:
missing a check for `err != nil` after the switch for an early escape before parsing the template